File: Fisica/src/Fisica/Models/EnderecoModel.cs
using Fisica.Domains;

namespace Fisica.Models
{
    public class EnderecoModel
    {
        public string Bairro { get; set; }
        public string Logradouro { get; set; }
        public int Numero { get; set; }
        public long CidadeId { get; set; }
        public CidadeModel Cidade { get; set; }
        public long? InstituicaoId { get; set; }
    }
}

File: Fisica/src/Fisica.Website/ClientApp/src/app/shared/models/endereco.model.ts
import { Cidade } from "./cidade.model";
import { Instituicao } from "./instituicao.model";

export class Endereco {
  constructor(
    public id?: number,
    public bairro?: string,
    public logradouro?: string,
    public numero?: number,
    public cidade?: Cidade,
    public cidadeId?: number,
    public institucao?: Instituicao
  ) { }
}
